components/invitation_modal/invite_as.tsx

```diff
@@ -25,102 +20,55 @@ export type Props = {
     setInviteAs: (inviteType: InviteType) => void;
     inviteType: InviteType;
     titleClass?: string;
-    inviteToTeamTreatment: InviteToTeamTreatments;
 }
 
 export default function InviteAs(props: Props) {
-    let title = (
-        <FormattedMessage
-            id='invite_modal.as'
-            defaultMessage='Invite as'
-        />
-    );
-    let control = (
-        <div>
-            <RadioGroup
-                onChange={(e) => props.setInviteAs(e.target.value as InviteType)}
-                value={props.inviteType}
-                id='invite-as'
-                values={[
-                    {
-                        key: (
-                            <FormattedMessage
-                                id='invite_modal.choose_member'
-                                defaultMessage='Member'
-                            />
-                        ),
-                        value: InviteType.MEMBER,
-                        testId: 'inviteMembersLink',
-                    },
-                    {
-                        key: (
-                            <span className='InviteAs__label'>
+    return (
+        <div className='InviteAs'>
+            <div className={props.titleClass}>
+                <FormattedMessage
+                    id='invite_modal.as'
+                    defaultMessage='Invite as'
+                />
+            </div>
+            <div>
+                <RadioGroup
+                    onChange={(e) => props.setInviteAs(e.target.value as InviteType)}
+                    value={props.inviteType}
+                    id='invite-as'
+                    values={[
+                        {
+                            key: (
                                 <FormattedMessage
-                                    id='invite_modal.choose_guest_a'
-                                    defaultMessage='Guest'
+                                    id='invite_modal.choose_member'
+                                    defaultMessage='Member'
                                 />
-                                <span className='InviteAs__label--parenthetical'>
-                                    {' - '}
+                            ),
+                            value: InviteType.MEMBER,
+                            testId: 'inviteMembersLink',
+                        },
+                        {
+                            key: (
+                                <span className='InviteAs__label'>
                                     <FormattedMessage
-                                        id='invite_modal.choose_guest_b'
-                                        defaultMessage='limited to select channels and teams'
+                                        id='invite_modal.choose_guest_a'
```

Comment:
nit: we can drop the `a` reference of the text

Reply:
they exist as a way to differentiate two phrases in the same area and are not related to the a/b testing.

Reply:
Gotcha, that's for clarifying!